tests/test_lsmop.py

```diff
@@ -16,13 +16,12 @@
 data = random.uniform(key, (100, 300), minval=0, maxval=upper)
 n, d = data.shape
 m = 3
-keys = jax.random.split(key, 16)
+# keys = jax.random.split(key, 16)
 
 
 def test_lsmop1():
-    global data, keys, d, m
     prob = LSMOP1(d=d, m=m)
-    state = prob.init(keys)
+    state = prob.init(key)
```

Comment:
When is `key` defined?

Reply:
at start, I define some global variables for all test functions.